ERLANG.md

```diff
@@ -161,13 +172,17 @@ And now, before we can run the `meta testing` suite, we need to generate the [`P
 ``` bash
 $ rebar3 dialyzer
 ```
+
 Now you can run:
+
 ``` bash
 $ rebar3 ct
 ```
 
 ### Data for hex.pm
+
 The project's app.src file should include the following tuples:
+
 ```erlang
   {maintainers,["Inaka"]},
```

Comment:
Do you mind removing this line one, too? It's deprecated.

Reply:
Not at all.

Reply:
71f6f65